react/components/MiniCartContent.js

```diff
@@ -113,59 +102,18 @@ class MiniCartContent extends Component {
     this.setState({ isUpdating })
   }
 
-  onUpdateItems = (id, quantity) => {
-    this.updateItemLoad(id, true)
-    const {
-      data: {
-        orderForm,
-        updateAndRefetchOrderForm,
-      },
-    } = this.props
-
-    const items = this.getGroupedItems()
-    const itemPayloadGrouped = items.find(item => item.id === id)
-    const itemsPayload = orderForm.items.filter(item => item.id === id)
-    let itemPayload = itemsPayload[0]
-    const index = orderForm.items.indexOf(itemsPayload[0])
-    const newQuantity = quantity - (itemPayloadGrouped.quantity - itemPayload.quantity)
-    const updatedItems = [
-      {
-        id: itemPayload.id,
-        index,
-        quantity: newQuantity,
-      },
-    ]
-
-    if (newQuantity <= 0) {
-      updatedItems[0].quantity = 0
-      itemPayload = itemsPayload[1]
-      updatedItems.push(
-        {
-          id: itemPayload.id,
-          index: orderForm.items.indexOf(itemPayload),
-          quantity: itemPayload.quantity + newQuantity,
-        }
-      )
-    }
-
-    updateAndRefetchOrderForm({
-      variables: {
-        orderFormId: orderForm.orderFormId,
-        items: updatedItems,
-      },
-    }).then(() => {
-      this.updateItemLoad(id, false)
-    })
-  }
-
-  sumOptionsPrice = (addedOptions = []) => {
-    return addedOptions.reduce((acc, option) => acc + option.sellingPrice * option.quantity, 0)
-  }
+  sumOptionsPrice = (addedOptions = []) =>
+    addedOptions.reduce(
+      (acc, option) => acc + option.sellingPrice * option.quantity,
```

Comment:
Repeated code for the sum price, no?

Reply:
Nop. Very similar, but this is `sellingPrice`. This is why I didn't change it

Reply:
`sumOptionsWith = prop => (addedOptions = []) =>...` or
`sumOptionsSellingPrice`